### DISCUSSION TITLE
New online service

### DISCUSSION THREAD

**Discussion by gridmarketsgithub** [lllyasviel/Fooocus#2735] [Show and tell]
GridMarkets is supporting Fooocus, Automatic1111 and ComfyUI for only $1/hr on an RTX A6000, so you can create 2048x2048 SDXL images with 40 steps in less than a minute.  You pay by the second and NOT for images, tokens, etc.  And the first 5hrs are free!

You also get 1Tb of persistent storage (up to 15 days as long as there's activity) for NO additional cost.  You can replicate your local set up and we preserve your progress and configurations so you'll save valuable time not having to re-setup all your workflows, images, etc. each time you start a machine.

See https://www.gridmarkets.com/stable-diffusion for more details.

**Comment by ILikeToasters** [comment]
One of the big complaints I see from people about many of the services if they are not generating and it is idle it uses time and costs money. Is that the same here?

**Reply by gridmarketsgithub** [reply]
You can set a budget to be Alerted (emailed) when you hit your limit, or just Terminate (stop) the machine, but we automatically save the state so you don't have to recreate the setup each time.  
![image](https://github.com/lllyasviel/Fooocus/assets/25288447/f04335f9-3ba2-4c88-8313-c7bf77497fbd)

[Signup](https://portal.gridmarkets.com/init/default/register?vertical=media) for 5 free hours to try it out.
